hazelcast-client/src/main/java/com/hazelcast/client/map/impl/nearcache/ClientHeapNearCache.java

```diff
@@ -85,11 +87,12 @@ public ClientHeapNearCache(String mapName, ClientContext context, NearCacheConfi
         if (inMemoryFormat != InMemoryFormat.BINARY && inMemoryFormat != InMemoryFormat.OBJECT) {
             throw new IllegalArgumentException("Illegal in-memory-format: " + inMemoryFormat);
         }
-        this.maxSize = nearCacheConfig.getMaxSize();
+        EvictionConfig evictionConfig = initDefaultMaxSize(nearCacheConfig.getEvictionConfig());
```

Comment:
This initializes the old default max size value, if no size was configured by the user.